book/04-git-server/sections/gitlab.asc

```diff
@@ -1,131 +1,132 @@
+Her proje ana sayfası, son etkinliği gösterir ve üst kısımdaki bağlantılar, proje dosyalarının ve işlem günlüğün görüntülerine yönlendirir.
 
-==== Working Together
+==== Birlikte Çalışma
 
-The simplest way of working together on a GitLab project is by giving another user direct push access to the Git repository.
-You can add a user to a project by going to the ``Members'' section of that project's settings, and associating the new user with an access level (the different access levels are discussed a bit in <<_gitlab_groups_section>>).
-By giving a user an access level of ``Developer'' or above, that user can push commits and branches directly to the repository with impunity.
+Bir GitLab projesinde birlikte çalışmanın en basit yolu, başka bir kullanıcıya Git reposuna doğrudan itme erişimi vermektedir.
+Bunu yapmak için, o projenin ayarlarında "Üyeler" bölümüne giderek yeni kullanıcıyı bir erişim seviyesi ile ilişkilendirebilirsiniz (farklı erişim seviyeleri konusuna <<_gitlab_groups_section>> bölümünde biraz girilmiştir).
+"Geliştirici" veya daha üst bir erişim seviyesi verilen bir kullanıcı o repoya gelişigüzel olarak doğrudan işlemler ve dallar ekleyebilir.
 
-Another, more decoupled way of collaboration is by using merge requests.
-This feature enables any user that can see a project to contribute to it in a controlled way.
-Users with direct access can simply create a branch, push commits to it, and open a merge request from their branch back into `master` or any other branch.
-Users who don't have push permissions for a repository can ``fork'' it (create their own copy), push commits to _that_ copy, and open a merge request from their fork back to the main project.
-This model allows the owner to be in full control of what goes into the repository and when, while allowing contributions from untrusted users.
+Daha bağımsız bir işbirliği yöntemi de "birleştirme isteklerini" kullanmaktır.
+Bu özellik, bir projeyi görebilen herhangi bir kullanıcının, denetimli bir şekilde katkıda bulunmasını sağlar.
+Doğrudan erişimi olan kullanıcılar basitçe bir dal oluşturabilir, işlemleri ona iter ve dallarından `master` veya başka bir dala birleştirme isteği açabilir.
+Depoya itme izinleri olmayan kullanıcılar onu "çatallayabilir" (kendi kopyalarını oluşturabilir), işlemleri bu kopyaya iter ve kendi çatalından ana projeye birleştirme isteği açabilir.
+Bu model "güvenilmeyen kullanıcılardan" katkı alınmasına olanak tanırken, sahibin (owner) de repoya neyin, ne zaman gireceği konusunda tam bir kontrol sahibi olmasını sağlar.
 
-Merge requests and issues are the main units of long-lived discussion in GitLab.
-Each merge request allows a line-by-line discussion of the proposed change (which supports a lightweight kind of code review), as well as a general overall discussion thread.
-Both can be assigned to users, or organized into milestones.
+Birleştirme istekleri ve sorunlar, GitLab'da uzun süreli tartışmanın ana birimleridir.
```

Comment:
issue terimi şimdiye kadar hep "sorun" olarak kullanılmış ama tekrar değerlendirip sonra toplu olarak değiştirmeyi tercih edebiliriz. örneğin şu an uğraştığımız projenin "issue"ları "terimler sözlüğü" ve "durum güncellemesi". genelde projelerde de iş akışı yeni bir feature ekleneceğinde önce onun için bir issue açılıp birine assign edilmesi şeklinde ilerliyor. hem bizim proje hem diğerleri için "sorun" olarak çevirdiğimizde bağlamından kopmuş oluyor

Reply:
Burada "iş paketi" veya benzeri bir terim kullanılabilir. İlkesel olarak, bağlam içerisinde ve durumsal yaklaşmak gerektiğini düşünüyorum. 

Tabi ki dikkatimden kaçan şeyler hatta hiç kullanmadığım için yabancı olduğum kavramlar olabilir. Çeviride ilerledikçi pek çok yeni şey öğreniyorum veya yeni durumlarla karşılaşınca daha önce kullandığım belli bir terimi, bir başkasıyla ikame etme fikri de gelişebiliyor.

Tüm çeviri bitince (hedefim en azından ilk 6 üniteyi çevirmek), ilk bölüm de dahil olmak üzere, hepsinin bir kez daha gözden geçirilmesi ve ortak bir terminolojiye bağlı olarak düzeltilmesi, faydalı olur diye düşünüyorum. "Güncel Terim Listesi" bu amaca hitabediyor aslında.

Bu arada terimlerle alakalı yorumlar yapılabilir, farklı öneriler getirilebilir.